### PR TITLE
Add links to Stack Exchange and Discord on the ‘create an issue’ page

### PR DESCRIPTION
**Pull request by steveluscher**
#### Problem

Sometimes people create issues when all they want is to ask a usage question.

#### Summary of Changes

I learned you can do this from the Typescript GitHub: https://github.com/microsoft/TypeScript/issues/new/choose

image

### CODE REVIEW

**Comment by jacobcreech** [comment]
> Can the type of person with something to discuss even find a public channel on Discord to do so, anymore?

There's some discussion in core-technology allowed. I would hope that if someone is opening an issue they'd be good enough for chatting about it. I'll defer to @t-nelson or @mvines though.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+contact_links:
+  - about: "Have a question about using this software, or about Solana in general? Post it on the Solana Stack Exchange."
+    name: Ask a Question
+    url: "https://solana.stackexchange.com/questions/ask"
+  - about: "Start or join a discussion on the Solana Tech Discord."
+    name: Start a Discussion
```
Comment (mvines) [inline .github/ISSUE_TEMPLATE/config.yml:6]: I’m ok with linking to discord with more detail about the kinds of discussions we hold within it:
No dev support. Solana protocol engineering, validator operations, and Solana Labs validator client engineering

**Comment by t-nelson** [comment]
Is there a way to preview changes?  Definitely agree with @mvines that any link out to discord needs the ground rules clearly definined.  Unclear how much formatting freedom we have with the description though

**Comment by t-nelson** [comment]
do we want to let this die?

**Pull request closed without merging**